src/components/generic/StackedCardsSwiper.vue

```diff
@@ -114,20 +128,58 @@
     mounted() {
       const middleCardIndex = Math.floor(this.slides.length / 2);
       this.swiper.slideTo(middleCardIndex);
+    },
+
+    methods: {
+      imageCSSVars(image) {
+        if (urlIsContentfulAsset(image.url)) {
+          return {
+            '--bg-img-small': `url('${optimisedSrcForContentfulAsset(image, { w: 245, h: 500, fit: 'fill' })}')`,
```

Comment:
These need to be compressed. Let's try quality level 80. Either add to each of these calls to `optimisedSrcForContentfulAsset`, or even consider making it a default in that function, thus:
```diff
diff --git a/src/plugins/contentful-utils.js b/src/plugins/contentful-utils.js
index 067fba4da..6792b3c45 100644
--- a/src/plugins/contentful-utils.js
+++ b/src/plugins/contentful-utils.js
@@ -1,4 +1,3 @@
-
 export function urlIsContentfulAsset(url) {
   const hostnameMatch = url.match(/\/\/([^/]+)\//);
   return hostnameMatch && (hostnameMatch[1] === 'images.ctfassets.net');
@@ -9,8 +8,11 @@ export function optimisedSrcForContentfulAsset(asset, params = {}) {
 
   // TODO: are optimisations possible on any other content types?
   if (asset.contentType === 'image/jpeg') {
-    params['fm'] = 'jpg';
-    params['fl'] = 'progressive';
+    params.fm = 'jpg';
+    params.fl = 'progressive';
+    if (!params.q) {
+      params.q = 80;
+    }
   }
 
   for (const key in params)  {
```